### PR TITLE
GET_VOLUME_STATS node capability added

### LINKED ISSUE #37: Node GetVolume stats implementation for node plugin
Reference: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/pkg/driver/node.go#L428

### PR DESCRIPTION
**Pull request by dharaneeshvrd**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Implemented NodeGetVolumeStats

**Which issue(s) this PR fixes**:

Fixes #37 

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: pkg/driver/node.go [hunk 1]
```diff
@@ -66,6 +67,7 @@ type nodeService struct {
 	driverOptions *Options
 	pvmInstanceId string
 	volumeLocks   *util.VolumeLocks
+	stats         StatsUtils
```
Comment (Madhan-SWE) [inline pkg/driver/node.go:70]: Don't we need to declare stats as a pointer?
it is assigned with address as below
```
stats:         &VolumeStatUtils{},
```

File: pkg/driver/node.go [hunk 2]
```diff
@@ -380,7 +383,72 @@ func (d *nodeService) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpu
...
+	volumePath := req.VolumePath
+	// return if path does not exist
+	if d.stats.IsPathNotExist(volumePath) {
+		return nil, status.Error(codes.NotFound, "VolumePath not exist")
+	}
+
+	// check if volume mode is raw volume mode
+	isBlock, err := d.stats.IsBlockDevice(volumePath)
+	if err != nil {
+		return nil, status.Error(codes.Internal, fmt.Sprintf("Failed to check volume %v is block device or not: %v", req.VolumeId, err))
```
Comment (Madhan-SWE) [inline pkg/driver/node.go:406]: change %v to %s for  volume id as it is string

**Review by Madhan-SWE — COMMENTED** [review]
Initial review

**Comment by Madhan-SWE** [comment]
@mkumatag I have added my comments.
See if you need any other changes.

File: pkg/driver/node.go (hunk 1 above)
Comment (dharaneeshvrd) [reply pkg/driver/node.go:70]: Here StatsUtils is an interface and receiver methods of this interface receives a pointer so used &VolumeStatUtils{}.

File: pkg/driver/node.go (hunk 2 above)
Comment (dharaneeshvrd) [reply pkg/driver/node.go]: Done.

**Pull request merged**